mods/ra/rules/ai.yaml

```diff
@@ -479,3 +479,135 @@ Player:
 				TargetMetric: Value
 				CheckRadius: 7c0
 
```

Comment:
It is also different to what we currently have.
Please remove the extra line here, or open a new PR adding lines between all of them then this can be rebased on top of that.